src/EPR.PRN.Backend.API/Repositories/Repository.cs

```diff
@@ -60,6 +60,30 @@ join ps in _eprContext.PrnStatus on p.PrnStatusId equals ps.Id
         return prnUpdateStatuses;
     }
 
+    public async Task<List<PrnStatusSync>> GetSyncStatuses(DateTime fromDate, DateTime toDate)
+    {
+        var result = await (from p in _eprContext.Prn
+                            join ps in _eprContext.PEprNpwdSync on p.Id equals ps.PRNId
+                            where p.StatusUpdatedOn >= fromDate && p.StatusUpdatedOn <= toDate
```

Comment:
this where should be on createdOn field on sync table not other way around. Can we confirm the result of this query my understanding is it will take everything from prn and null if don't find anything on sync we don't want that we want to retrieve all the record on sync table within date range and details of it from prn table